### ISSUE TITLE
Problems with testing and visualization on the scannet dataset

### ISSUE THREAD

**Issue by gravity-123** [SamsungLabs/fcaf3d#54]
Hello, I want to run this model. I run the code on the mmdetection3d-1.0.0rc5 version.
I want to test scense0004_00 and  scense0009_00 in the scannet dataset using the weights you provided. The weights are downloaded in the red box in the figure below.
![80491248ca14a64874af7ad8c37e5ad3](https://github.com/user-attachments/assets/d4455b11-60f8-43d3-a457-38852e8e013f)
I used the visualization command `python tools/test.py configs/fcaf3d/fcaf3d_8x2_scannet-3d-18class.py work_dirs/fcaf3d_2xb8_scannet-3d-18class/20211007_144747.pth --eval mAP --show --show-dir work_dirs/fcaf3d_2xb8_scannet-3d-18class` ,but the test results and visualization results are as shown below.
![2024-10-16 11-33-45屏幕截图](https://github.com/user-attachments/assets/c5676ad3-7a2c-449f-9e07-366a987ffa1b)
![2024-10-16 11-37-02屏幕截图](https://github.com/user-attachments/assets/e9f956be-c2ad-4c67-9148-0004c538e1cd)
Due to my limited storage space, I made a very small training set and validation set.
![2024-10-16 13-19-25屏幕截图](https://github.com/user-attachments/assets/bdf44aba-c275-4081-9bf2-229f9484a762)
![2024-10-16 13-20-09屏幕截图](https://github.com/user-attachments/assets/aaa5f2ee-1f0a-4c7f-86a6-d43827b32aab)
I ran the training command `bash tools/dist_train.sh configs/fcaf3d/fcaf3d_8x2_scannet-3d-18class.py 1` and visualization command `python tools/test.py configs/fcaf3d/fcaf3d_8x2_scannet-3d-18class.py work_dirs/fcaf3d_8x2_scannet-3d-18class/latest.pth --eval mAP --show --show-dir work_dirs/fcaf3d_8x2_scannet-3d-18class`, but the test results and visualization results are as shown below.
![2024-10-16 11-34-35屏幕截图](https://github.com/user-attachments/assets/fa1343bc-88fe-46ce-8f5d-635f71016dc1)
![2024-10-16 11-37-57屏幕截图](https://github.com/user-attachments/assets/4cd7ed21-42eb-491d-a6e2-0d4d35b868fb)
In addition, I also changed score_thr=0.2, but I changed the score_thr parameter in configs/_base_/models/fcaf3d.py. Although it is different from the directory in your repository, I think I should have made no mistakes.
So I don’t know now whether there is a problem with the weights, the dataset, mmdetection3d-1.0.0rc5, the meshlab settings, or other problems. I hope you can help me take a look when you are free. Thank you very much.
If I want to get the effect in the link, how can I do it? https://github.com/SamsungLabs/fcaf3d/issues/31#issuecomment-1108180811

**Comment by filaPro** [comment]
Hi @gravity-123 ,
> I run the code on the mmdetection3d-1.0.0rc5 version
As I remember following our Dockerfile, you don't need to install mmdetection3d, if you are installing fcaf3d. Or if you are using mmdetection3d you should load their checkpoint for FCAF3D, not from this repo.

**Comment by gravity-123** [comment]
Thank you, I have solved the problem now.
